app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -380,9 +381,13 @@ protected void onStop() {
 
         if (SettingsStore.getInstance(this).getCylinderDensity() > 0.0f) {
             TelemetryWrapper.queueCurvedModeActiveEvent();
+            GleanMetricsService.curveModeActiveEvent(true);
```

Comment:
Does this buy you anything over just calling the Glean APIs? (and in other places as well)

Reply:
It's just for easy maintenance.